### ISSUE TITLE
Allow to pass options to default markdown renderer (ie: ability to pass remark plugins options)

### ISSUE THREAD

**Issue by MoOx** [phenomic/phenomic#668]
Ref #666

**Comment by thangngoc89** [comment]
User can pass in a custom `mdify` function in phenomic-loader-plugin-markdown-transform-body-property-to-html (the default one is [here](https://github.com/MoOx/phenomic/blob/13a8353d0c0c73752eca64b1b8c915f8701972c1/src/phenomic-loader-plugin-markdown-transform-body-property-to-html/index.js#L9))

Usage: 

```js
// webpack.config.js

const mdify = (text) => (
    remark()
    .use('remark-html')
    .process(text)
    .toString()
)
{
  phenomic: {
    plugins: [ ...phenomicLoaderPresets.markdown(mdify) ]
  }
}
```

**Comment by MoOx** [comment]
Or maybe just

```js
export const makeConfig = () => 
{
  ...
  phenomic: {
    plugins: phenomicLoaderPresets.markdown((remark) => {
      remark.use(html)
    })
  }
}
```

@DavidWells how does this looks for you?

We could think of another way to allow flag to disable default plugins, but I am afraid it will be more complicated...

**Comment by DavidWells** [comment]
Yeah definitely don't want to make things complicated.

Trying to find any info on how google indexes `src code` snippets in posts but coming up blank. Maybe I'm just paranoid here. 

It would appear that http://putaindecode.io/ stuff shows up when I search for specific `src` code so I'd guess this isn't a real issue https://www.google.com/search?q=%22%3Ch1+class%3D%22titleColor__abc5436+bigTitle__def6547%22%3E%3C%2Fh1%3E%22&oq=%22%3Ch1+class%3D%22titleColor__abc5436+bigTitle__def6547%22%3E%3C%2Fh1%3E%22&aqs=chrome..69i57.855j0j4&sourceid=chrome&ie=UTF-8

**Comment by MoOx** [comment]
> It would appear that http://putaindecode.io/ stuff shows up when I search for specific src code so I'd guess this isn't a real issue

Indeed, the reference is post is about CSS modules that show correctly a example of an hashed class name :D

Cool to know you are happy with the default renderer :)

**Comment by bloodyowl** [comment]
Going to be fixed with #925 (moving to user-land through plugins)

**Comment by MoOx** [comment]
Coming in next release of v1
